tox.ini

```diff
@@ -502,7 +502,7 @@ commands =
   codespell
 
 [testenv:lint]
-basepython: python3.10
+basepython: python3.9
```

Comment:
Is this change needed?

Reply:
Yes, seems related, `lint` fails without this change